Thinktecture.Relay.OnPremiseConnector/OnPremiseTarget/AcknowledgmentMode.cs

```diff
@@ -0,0 +1,27 @@
+namespace Thinktecture.Relay.OnPremiseConnector.OnPremiseTarget
+{
+	/// <summary>
+	/// This is the pendant for the server enum AcknowledgementMode and should be kept separate
+	/// </summary>
+	public enum AcknowledgmentMode
+	{
+		/// <summary>
+		/// Requests in the message queue will be automatically acknowledged by
```

Comment:
`Requests in the message queue will be automatically acknowledged by the on-premise before contacting the local target`